vizro-core/mkdocs.yml

```diff
@@ -30,9 +30,11 @@ nav:
           - Assets: pages/user-guides/assets.md
       - ACTIONS:
           - Actions: pages/user-guides/actions.md
-      - DATA CONNECTIONS:
-          - Data: pages/user-guides/data.md
-          - Integrations: pages/user-guides/integration.md
+      - DATA:
+          - Overview: pages/user-guides/data.md
+          - Static data: pages/user-guides/static-data.md
+          - Dynamic data: pages/user-guides/dynamic-data.md
+          - Kedro Data Catalog: pages/user-guides/kedro-data-catalog.md
```

Comment:
As mentioned above, I think this is too many pages for too little content (albeit being some of the most exciting content 😍 )